.travis.yml

```diff
@@ -52,8 +52,9 @@ script:
   - |
       if [[ $TESTS == 'blocks-examples' ]]; then
         cd $TRAVIS_BUILD_DIR
-        git clone https://github.com/mila-udem/blocks-examples.git
+        git clone https://github.com/laurent-dinh/blocks-examples.git
```

Comment:
Forgive me, I am blind! All right, then let's merge this PR. Can you please remove the hacks from .travis.yml? I will press the button then.